### PR TITLE
Fix Issue #2

### PR DESCRIPTION
**Pull request by basictheprogram**
Thank you for the quick fix to Issue #2 ! 

Here's my PR. Same fix you did so you can cherry pick and ignore that change. But I added some nose unit tests to confirm the regex is correct.

I did not see a license so I put the GPL v2 on the tests. Let me know if you would like a difference license.

I'm hoping you and the other contributors will pick up on the nose tests and start adding more unit tests to this module.

Thanks.

### CODE REVIEW

**Comment by basictheprogram** [comment]
In the setup.py I see http://www.opensource.org/licenses/bsd-license.php, I'll change the license on the test suite.

File: README.md [hunk 1]
```diff
@@ -63,6 +63,14 @@ There is nothing special to configure on PyFileMaker. Just make sure the
 PyFileMaker directory, which contains the file FMPro.py/FMServer.py file is
 somewhere in your Python path so that Python knows where to find the module.
 
+You can install v2.6 from pypi using pip
```
Comment (kawa-marcin) [inline README.md]: This is old version and does not work with the new FileMaker, also the methods described in the documentation in this project (like executing scripts or parsing response to JSON format) doesn't exist in the old library. I think we shouldn't encourage people to install the old version as it's really out of date.

Comment (basictheprogram) [inline README.md]: Agreed. But 2.6 is the only version available via pypi. I did a little work and have 3.0.2.1 setup on the testpypi:

https://testpypi.python.org/pypi/PyFileMaker/3.0.2.1

Finish off the PR and then you can put the real 3.0.2.1 release up on pypi?

Don't know what to do about the README. Github want's markdown but pypi wants text.

**Comment by kawa-marcin** [comment]
Thanks you put the package on pypi. I've never did before but I quick check it and apparently you can use the same md file like in github

You just need to include this in the `setup.cfg`

```
[metadata]
description-file = README.md
```

Please see http://peterdowns.com/posts/first-time-with-pypi.html

**Comment by basictheprogram** [comment]
Does it render the page nicely?

https://testpypi.python.org/pypi/PyFileMaker/3.0.2.1

The README.md looks terrible at the testing site.

**Pull request merged**